### DISCUSSION TITLE
Migrating a non-null column to be nullable

### DISCUSSION THREAD

**Discussion by lpotthast** [SeaQL/sea-orm#824] [Q&A] [answered]
In a prior migration, I initially defined a column like this
```
.col(ColumnDef::new(course_meeting::Column::End.date_time().not_null())
```

I realized that the column has to accept null.

So I added the following in a new migration
```
.modify_column(
    ColumnDef::new(course_meeting::Column::End).date_time(),
)
```
hoping that this would remove the "not null" flag.

But after running the migration, the column is still reported as not-nullable.

Even though there is a `ColumnSpec::Null`, it seems that there is no `.null()` or `.nullable()` function to be called on the `ColumnDef` so that this spec would be pushed to the internal vec...

Have I done something wrong here or is there an easier way of making a column nullable with a migration?

**Comment by billy1624** [comment, marked as answer]
Hey @inf0rm4tik3r, good questions! Yeah, I think we should add a `.null()` method in `ColumnDef`.

CC @ikrivosheev

**Reply by ikrivosheev** [reply]
Ah, yes! Good solution!

**Reply by ikrivosheev** [reply]
I will add it today and make PR.

**Reply by ikrivosheev** [reply]
@inf0rm4tik3r @billy1624 done, PR: https://github.com/SeaQL/sea-query/pull/365
